service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/placement/PlacementSubmittedControllerTest.java

```diff
@@ -360,6 +374,58 @@ void shouldNotMakePaymentWhenPlacementIsUpdated() {
         verifyNoInteractions(notificationClient, paymentApi, coreCaseDataApi);
     }
 
+    @Nested
+    class TestSealing {
+
+        private final Document sealedDocument = testDocument();
+        private final DocumentReference sealedApplication = buildFromDocument(sealedDocument);
```

Comment:
`sealedApplication` is never used

Reply:
updated the unit test, some changes are discarded accidentally when updating from master